examples/llm_pretrain/nanogpt_pretrain.yaml

```diff
@@ -6,6 +6,7 @@ step_scheduler:
   local_batch_size: 2
   ckpt_every_steps: 2000
   num_epochs: 1
+  max_steps: 9500
```

Comment:
Noting that I chose this `max_steps` value based on other similar-looking pretraining examples. Happy to change to something else if folks prefer.